wgengine/netstack/netstack.go

```diff
@@ -560,6 +563,7 @@ func (ns *Impl) forwardTCP(client *gonet.TCPConn, clientRemoteIP netaddr.IP, wq
 	go func() {
 		select {
 		case <-notifyCh:
+			ns.logf("[v2] netstack: forwardTCP notifyCh fired; canceling context for %s", dialAddrStr)
```

Comment:
Should this be in if block with debugNetstack?

Reply:
Yeah, guess so. Done.